plotnine/tests/test_facets.py

```diff
@@ -91,6 +91,17 @@ def test_facet_grid_scales_free_y():
          + theme(panel_spacing_x=0.3))
     assert p == 'facet_grid_scales_free_y'
 
+def test_facet_grid_scales_free_y_formula_dot_notation():
+    p = (g
+         + facet_grid('. ~ var1>2', scales='free_y')
```

Comment:
I do not see it, but there is a funny [character](https://travis-ci.org/has2k1/plotnine/jobs/357149958#L1129) on this line. I think you need to make sure that all the spaces are true ascii spaces.